tests/test-landpatterns.stanza

```diff
@@ -0,0 +1,33 @@
+#use-added-syntax(jitx, tests)
+defpackage ocdb/tests/landpatterns:
+  import core
+  import collections
+  import jitx
+  import jitx/commands
+  import ocdb/box-symbol
+  import ocdb/defaults
+  import ocdb/land-patterns
+
+deftest(ocdb, landpatterns, harness) four-sided-landpattern:
+  pcb-pad P:
+    val shape_ = RoundedRectangle(1.0 - 0.08, 0.25, 0.25)
+    shape = shape_
```

Comment:
Is `shape = RoundedRectangle(1.0 - 0.08, 0.25, 0.25)` possible?

Reply:
yes but I'm using it to add the soldermask below

Reply:
and you cannot access to the `shape`, right? `offset(shape, 0.08)`

Reply:
That requires introspection on `self` which is a little hairy and I did not want to have it impact this test

Reply:
fair enough!